docs/kafka/access-mgmt-kafka/README.adoc

```diff
@@ -323,7 +323,7 @@ If you selected a permission entry that applies to a specific account, you can d
  ** For the account that you want to delete, select the options icon (three vertical dots) for that entry and click *Delete*.
 
 +
-IMPORTANT: If you delete a user account or service account, you should also delete any ACL permissions associated with that account. If you don't delete unused ACL permissions, then a future account with the same ID of a previously deleted account could inherit the ACL permissions and have automatic access to a Kafka instance.
+IMPORTANT: If you delete a user account or service account, you also need to delete any ACL permissions associated with that account. If you don't delete unused ACL permissions, then a future account with the same ID of a previously deleted account could inherit the ACL permissions and have automatic access to a Kafka instance.
```

Comment:
This is definitely an improvement on _should_. But IBM actually recommends _must_ instead of _need to_ where possible: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#n